build.properties

```diff
@@ -16,7 +16,7 @@
 #      vivo.core.dir = ../vivo
 #      vivo.core.dir = /usr/local/vitro/trunk
 #      nihvivo.dir = ../vivo-rel-1.2
-vivo.core.dir = ./vivo
+vivo.core.dir = /Users/Hao/Projects/VIVO/
```

Comment:
Should not have committed and pushed this file. Please change back. In the future you can change this file locally without adding, committing and pushing the change